persistence/sql/persister.go

```diff
@@ -56,7 +56,11 @@ func (p *Persister) MigrateDown(ctx context.Context, steps int) error {
 }
 
 func (p *Persister) MigrateUp(ctx context.Context) error {
-	return errors.WithStack(p.mb.Up())
+	if err := p.mb.Up(); err != nil {
+		return errors.WithStack(err)
+	}
+
+	return identity.GuaranteeCredentialTypes(p.c)
```

Comment:
Shouldn't this be just it's own SQL Migration? Why is it handled in the code?

Reply:
I wanted to make sure that we don't have to duplicate the data into sql and forget to update it there. But it makes more sense, if one would be renamed we would have to do it in a migration. I will change it and add tests to check whether the migration adds the correct data.

Reply:
Ooops missed that!

Reply:
Missed what?

Reply:
Sorry I got confused with the notification emails. Yes please do it as a migration. It is cleaner and allows for updates.